internal/util/http.go

```diff
@@ -26,6 +27,14 @@ import (
 	"github.com/coreos/ignition/internal/version"
 )
 
+const (
+	maxAttempts = 15
+)
```

Comment:
Should this be a property of a client in case we want to make it configurable later?